### PR TITLE
Use lib-upgrader for the CLI

### PR DESCRIPTION
**Pull request by jfmengels**
Use lib-upgrader for the CLI
As mentioned in #8, I wanted to move the CLI into a separate project, so that's what I did.

Here's the link to https://github.com/jfmengels/lib-upgrader

I've added a few things in there too, including (off the top of my head):
- Default selection of the `to` last version
- Do not show versions before the selected `from` version
- Support for --from and --to
- Prints a top at then end of the command that says

```
For similar projects, you may want to run the following command (based on the answers to the questions):
    ava-codemods --from 0.0.0 --to 4.0.0 "test.js" "test-*.js" "test/**/*.js"
```
- Uses `jscodeshift` runner (@sindresorhus you were right, there was an API, though not made officially public. I'll contact the maintainers at some point to know whether they can make it official)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,33 +18,23 @@
...
   "files": [
     "cli.js",
-    "cli-utils.js",
-    "codemods.json",
+    "releases.json",
     "lib"
   ],
   "keywords": [
     "codemod",
-    "ava",
```
Comment (LinusU) [inline package.json:27]: The `ava` keyword should probably stay though?

Comment (jfmengels) [inline package.json:27]: Right ;)

**Comment by jamestalmage** [comment]
I am going to close in favor of waiting for https://github.com/jfmengels/lib-upgrader/issues/2#issuecomment-209451744 to happen.

I've got mocha support in the works (#20), and I think it will be a lot easier to reconcile that with `lib-upgrader` once the API is made a bit more flexible.

**Comment by jfmengels** [comment]
:+1: Have to motivate myself to work on it at some point...

**Comment by jamestalmage** [comment]
Cool to delete this branch, or you want to keep it around for reference?

**Comment by jfmengels** [comment]
Pushed it to my fork :+1:

**Pull request closed without merging**